specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-11-01/schemaregistry.json

```diff
@@ -0,0 +1,441 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Schema Registry",
+    "version": "2021-11-01",
+    "description": "Azure Schema Registry is as a central schema repository, with support for versioning, management, compatibility checking, and RBAC."
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": true,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/$schemagroups/getSchemaById/{schema-id}": {
```

Comment:
The "$" character is necessary because of a technical limitation on the service endpoint. The schema registry currently is hosted on the EventHub host platform, and in the Event Hub service, the path of all endpoint actually are user usable for entity name - the implication here means that we cannot just call our entity type "schemagroups" because it can actually clash with some user created resources with the same name.

So it is EventHub's design choice to have all system name to prefix with "$" (customer cannot create resource with "$" in them)